easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.3.4-intel-2017a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFmpeg'
+version = '3.3.4'
+
+homepage = 'https://www.ffmpeg.org/'
+description = """A complete, cross-platform solution to record, convert and stream audio and video."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://ffmpeg.org/releases/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['5ef5e9276c311c74ab2e9d301c2d7ee10e1f2cbd758c6f13d6cb9514dffbac7e']
+
+dependencies = [
+    ('NASM', '2.12.02'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+    ('x264', '20170913'),
```

Comment:
@boegel any reason why `LAME` and `x265` were removed from the dependencies? Upstream changes?

Reply:
Hmm, that wasn't the intention... Will fix.